lib/api/initiateMultipartUpload.js

```diff
@@ -64,6 +64,13 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
     // multipart upload object with a key containing the splitter.
     const websiteRedirectHeader =
         request.headers['x-amz-website-redirect-location'];
+    if (request.headers['x-amz-storage-class'] &&
```

Comment:
```suggestion
    const storageClasses = ['STANDARD', 'STANDARD_IA', 'REDUCED_REDUNDANCY'];
    const requestStorageClass = request.headers['x-amz-storage-class'];
    if (requestStorageClass && !storageClasses.includes(requestStorageClass) {
```
Should this not includes other valid storage classes as updated in citadel?

Reply:
Hi Naren, closing this, as discussed in PM, as the changes were made in other commits. I will make sure to squash before merging.